core/src/core/verifier_cache.rs

```diff
@@ -28,20 +28,20 @@ pub trait VerifierCache: Sync + Send {
 	/// that have not yet been verified.
 	fn filter_kernel_sig_unverified(&mut self, kernels: &[TxKernel]) -> Vec<TxKernel>;
 	/// Takes a vec of tx outputs and returns those outputs
-	/// that have not yet had their rangeproofs verified.
-	fn filter_rangeproof_unverified(&mut self, outputs: &[Output]) -> Vec<Output>;
+	/// that have not yet been verified.
+	fn filter_output_unverified(&mut self, outputs: &[Output]) -> Vec<Output>;
 	/// Adds a vec of tx kernels to the cache (used in conjunction with the the filter above).
 	fn add_kernel_sig_verified(&mut self, kernels: Vec<TxKernel>);
 	/// Adds a vec of outputs to the cache (used in conjunction with the the filter above).
-	fn add_rangeproof_verified(&mut self, outputs: Vec<Output>);
```

Comment:
nit: The names and comments of these 2 functions were more accurate before. There's no need to rebrand all of this as "output verification." It's still rangeproof verification imho. It just includes the output commitment as part of the hash.